e2e/pages/events/manageOrders.page.js

```diff
@@ -283,16 +283,29 @@ const selectCafcassRegion = region => {
   I.click(`${section4.cafcassRegion.group}-${region}`);
 };
 
-const selectEnglandOffice= office => {
+const selectEnglandOffice = office => {
   I.selectOption(section4.englandOffices, office);
 };
 
+const linkApplication = async (applicationId) => {
```

Comment:
minor: as an e2e test, is maybe more readeable chosing the option via value rather than key?

Reply:
Good shout. Done.